src/sentry/stacktraces.py

```diff
@@ -0,0 +1,195 @@
+from __future__ import absolute_import
+
+import logging
+
+from sentry.models import Project
+from sentry.utils.safe import safe_execute
+
+
+logger = logging.getLogger(__name__)
+
+
+class StacktraceInfo(object):
+    """A wrapper around a stacktrace that was extracted from event
+    data.
+    """
+
+    def __init__(self, stacktrace, container=None, platforms=None):
+        self.stacktrace = stacktrace
+        self.container = container
+        self.platforms = platforms
+
+
+class StacktraceProcessor(object):
+
+    def __init__(self, data, stacktrace_infos):
+        self.data = data
+        self.stacktrace_infos = stacktrace_infos
+        self.project = Project.objects.get_from_cache(
```

Comment:
memo to myself: this does not do what i thought it does. it gets it from memcache and not in memory caching. Should pass this in then.